### ISSUE TITLE
Can't make VideoFileClip 'utf8'

### ISSUE THREAD

**Issue by ArchiZT** [Zulko/moviepy#169]
```
'utf8' codec can't decode byte 0xce in position 1873: invalid continuation byte
Exception AttributeError: "VideoFileClip instance has no attribute 'reader'" in <bound method
VideoFileClip.__del__ of <moviepy.video.io.VideoFileClip.VideoFileClip instance at 0x02D9F440>> ignored
```

**Comment by Zulko** [comment]
I can't understand what the problem is. Can you paste the code that produced that error ?

**Comment by ArchiZT** [comment]
epStr ='63'
path = "C:/Users/archiz/Desktop/today/_MONTAGE/"+epStr+"/"
print path
plays = os.listdir(path)
print plays
clipArr = []
for play in plays:
    print play
    try:
        clipArr.append( VideoFileClip( path+play ).fx( vfx.resize, width=2128, height=872) )
    except Exception, e:
        print e

final_clip = concatenate_videoclips(clipArr)
final_clip.write_videofile("C:/Users/archiz/Desktop/today/_MONTAGE/"+epStr+"/MONTAGE_"+epStr+".avi", codec='libx264', fps=24)

**Comment by ArchiZT** [comment]
it just lists all video files in folder and making montage out of them, nothing special, but one of the files is producing this error when I'm trying to make VideoFileClip out of it.

**Comment by Zulko** [comment]
That's a little clearer. Can you post the whole error message ?

**Comment by ArchiZT** [comment]
Traceback (most recent call last):
  File "C:/Users/archiz/PycharmProjects/untitled/temspsa.py", line 13, in <module>
    clipArr.append( VideoFileClip( path+play ).fx( vfx.resize, width=2128, height=872) )
  File "C:\Python27\lib\site-packages\moviepy-0.2.2-py2.7.egg\moviepy\video\io\VideoFileClip.py", line 55, in __init__
    reader = FFMPEG_VideoReader(filename, pix_fmt=pix_fmt)
  File "C:\Python27\lib\site-packages\moviepy-0.2.2-py2.7.egg\moviepy\video\io\ffmpeg_reader.py", line 32, in __init__
    infos = ffmpeg_parse_infos(filename, print_infos, check_duration)
  File "C:\Python27\lib\site-packages\moviepy-0.2.2-py2.7.egg\moviepy\video\io\ffmpeg_reader.py", line 235, in ffmpeg_parse_infos
    infos = proc.stderr.read().decode('utf8')
  File "C:\Python27\lib\encodings\utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xce in position 1873: invalid continuation byte
Exception AttributeError: "VideoFileClip instance has no attribute 'reader'" in <bound method VideoFileClip.__del__ of <moviepy.video.io.VideoFileClip.VideoFileClip instance at 0x02D29B98>> ignored

**Comment by ArchiZT** [comment]
If I delete vfx.resize, It prints this.

Traceback (most recent call last):
  File "C:/Users/archiz/PycharmProjects/untitled/temspsa.py", line 13, in <module>
    clipArr.append( VideoFileClip( path+play ) )
  File "C:\Python27\lib\site-packages\moviepy-0.2.2-py2.7.egg\moviepy\video\io\VideoFileClip.py", line 55, in __init__
    reader = FFMPEG_VideoReader(filename, pix_fmt=pix_fmt)
  File "C:\Python27\lib\site-packages\moviepy-0.2.2-py2.7.egg\moviepy\video\io\ffmpeg_reader.py", line 32, in __init__
    infos = ffmpeg_parse_infos(filename, print_infos, check_duration)
  File "C:\Python27\lib\site-packages\moviepy-0.2.2-py2.7.egg\moviepy\video\io\ffmpeg_reader.py", line 235, in ffmpeg_parse_infos
    infos = proc.stderr.read().decode('utf8')
  File "C:\Python27\lib\encodings\utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xce in position 1873: invalid continuation byte
Exception AttributeError: "VideoFileClip instance has no attribute 'reader'" in <bound method VideoFileClip.__del__ of <moviepy.video.io.VideoFileClip.VideoFileClip instance at 0x02F04440>> ignored

**Comment by Zulko** [comment]
Looks like some information in the file can't decode to utf8. Can you confirm that it's just some files, and that other files work ? If so, can you share the file that doesn't work ? Also, what Python version are you using ?

**Comment by ArchiZT** [comment]
Yes, other files work just fine. I'm sorry I can't send you this particular file, but maybe tomorrow I'll ask person who sent me the file to make test video with codec that he is using and will send to you. I'm using Python 2.7

**Comment by ArchiZT** [comment]
Here is a test sample https://www.dropbox.com/s/ud3jnga6lzkxqox/test%20video.rar?dl=0

**Comment by ArchiZT** [comment]
I guess it has to be something with making .mov files on russian windows.

**Comment by ArchiZT** [comment]
if I convert such video with ffmpeg (ffmpeg -i test.mov -o test.avi) moviepy don't make any errors. Maybe convert such videos to some standart extension when moviepy concatenating them?

**Comment by earney** [comment]
can we close this issue?
From the comments it seems the user used ffmpeg to convert the video clips that were causing issues, and then rerunning the script.
Is this correct?

**Comment by mylyu** [comment]
I met the same problem. Someone has proposed to correct this by using
`
decode('utf8','ignore')`

https://blog.51cto.com/cfy10/2391370

**Comment by delbos-d** [comment]
Hi,

I have the same trouble, so I investigate and I found the line, in the moviepy lib, it's the file video/io/ffmpeg_reader.py line 263.
The error come from the line error.decode('utf8'),
The error content is:
b"ffmpeg version 4.1-static https://johnvansickle.com/ffmpeg/  Copyright (c) 2000-2018 the FFmpeg developers
  built with gcc 6.3.0 (Debian 6.3.0-18+deb9u1) 20170516
  configuration: --enable-gpl --enable-version3 --enable-static --disable-debug --disable-ffplay --disable-indev=sndio --disable-outdev=sndio --cc=gcc-6 --enable-fontconfig --enable-frei0r --enable-gnutls --enable-gray --enable-libaom --enable-libfribidi --enable-libass --enable-libvmaf --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-librubberband --enable-libsoxr --enable-libspeex --enable-libvorbis --enable-libopus --enable-libtheora --enable-libvidstab --enable-libvo-amrwbenc --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzimg
  libavutil      56. 22.100 / 56. 22.100
  libavcodec     58. 35.100 / 58. 35.100
  libavformat    58. 20.100 / 58. 20.100
  libavdevice    58.  5.100 / 58.  5.100
  libavfilter     7. 40.101 /  7. 40.101
  libswscale      5.  3.100 /  5.  3.100
  libswresample   3.  3.100 /  3.  3.100
  libpostproc    55.  3.100 / 55.  3.100
  Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'e00f5d50d5c444a8894d119dd7f879bf.mp4':
  Metadata:
    major_brand     : mp42
    minor_version   : 1
    compatible_brands: mp42mp41
    creation_time   : 2017-09-11T09:44:59.000000Z
  Duration: 00:00:20.40, start: 0.000000, bitrate: 10186 kb/s
    Stream #0:0(eng): Video: h264 (Constrained Baseline) (avc1 / 0x31637661), yuv420p(tv, bt709), 1920x1080, 9926 kb/s, 25 fps, 25 tbr, 25 tbn, 50 tbc (default)
    Metadata:
      creation_time   : 2017-09-11T09:44:59.000000Z
      handler_name    : Module de gestion vid\x8eo
    Stream #0:1(eng): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 256 kb/s (default)
    Metadata:
      creation_time   : 2017-09-11T09:44:59.000000Z
      handler_name    : Module de gestion sonore
      At least one output file must be specified
"

Indeed correct the line 263 with the line: `infos = error.decode('utf8','ignore')` fix the bug @Zulko can you investigate this error? I tried a basic `VideoFileClip("file.mp4")` or `VideoFileClip("file.mov")` but the error is the same.

The bug is the same in 1.0.0 and 1.0.1.

Thanks a lot!

**Comment by tburrows13** [comment]
Fixed in #959
